pkgs/by-name/la/landrop/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  qt6,
+  libsodium,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "landrop";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "LANDrop";
+    repo = "LANDrop";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-IwtphjMSa0e2mO5C4zHId48SUpT99sXziZzApnSmvrU=";
+  };
+
+  sourceRoot = "${finalAttrs.src.name}/LANDrop";
+
+  postPatch = ''
+    substituteInPlace LANDrop.pro \
+        --replace-fail '$$(PREFIX)' '$$PREFIX'
+  '';
+
+  nativeBuildInputs = [
+    qt6.qmake
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs = [ libsodium ];
```

Comment:
Do we need `qt6.qtbase` and `qt6.qtwayland`? I saw in the description that it comes with a GUI. It's possible that you installed these packages in profiles so that it works in your environment.

Reply:
I think the basic `qt6.qtbase` dependencies are propagated when using the `qt6.qmake` hook. And AFAICT the entire gui is opened/controlled by the taskbar applet/icon.

I haven't actually tried it on my phone, but I'll do that now.